src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/CategoriesDocument.cs

```diff
@@ -83,7 +88,7 @@ public CategoriesDocumentFormatter GetFormatter()
 
         public void Save(XmlWriter writer)
         {
-            GetFormatter().WriteTo(writer);
+            GetFormatter().WriteToAsync(writer);
```

Comment:
This will keep going in the background writing to the XmlWriter so the caller might close the writer before we're done.

Reply:
Good catch. What I changed was just the name of the method. The method was named `WriteTo`, but it was an async method. 

I'll fix this.